main.go

```diff
@@ -265,6 +271,23 @@ func main() {
 	<-ctx.Done()
 }
 
+func getSriovTokenServerChainElement(ctx context.Context) (tokenServer networkservice.NetworkServiceServer) {
+	sriovTokens := tokens.FromEnv(os.Environ())
+	switch len(sriovTokens) {
+	case 0:
+		tokenServer = null.NewServer()
+	case 1:
+		var tokenKey string
+		for tokenKey = range sriovTokens {
+			break
+		}
+		tokenServer = token.NewServer(tokenKey)
+	default:
+		log.FromContext(ctx).Fatalf("endpoint must be configured with none or only one sriov resource")
```

Comment:
Consider using injecterror.NewServer() for this branch

Reply:
@denis-tingaikin
Is there any reason to postpone error from the application start time to the first Request time? Endpoint will be invalid in such case, why do we need to even start it?

Reply:
It was needed if we go with switchcase, but I'm fine with current version.